src/main/java/com/commercetools/sync/sdk2/products/utils/VariantReferenceResolutionUtils.java

```diff
@@ -86,57 +85,64 @@ static List<PriceDraft> mapToPriceDrafts(
    * @return a {@link List} of {@link Attribute} that has all product references with keys replacing
    *     the ids.
    */
+  // TODO: Is this method required in SDK2 or not??
   @Nonnull
   static List<Attribute> replaceAttributesReferencesIdsWithKeys(
```

Comment:
I see this method is not being used. In the old code it was used in `mapToProductVariantDraft` in this class. Please check if it's not necessary anymore, then just remove it and also its tests.

Reply:
[fb24318](https://github.com/commercetools/commercetools-sync-java/pull/941/commits/fb24318c8ec0fe674e27c1d9e3e15b0601a30671)